packages/portal/src/components/stories/StoriesInterface.vue

```diff
@@ -38,6 +38,7 @@
       <StoriesFeaturedCard
         v-if="showFeaturedStory"
         :featured-story="featuredStory"
+        :image="feature"
```

Comment:
What is this for?

Reply:
nothing anymore. I was trying to re-organise the prop data, but threw that out. Forgot to remove this.